src/formGroup.vue

```diff
@@ -32,6 +32,12 @@ export default {
 	name: "form-group",
 	components: fieldComponents,
 	mixins: [formMixin],
+	beforeDestroy() {
+		// if deleteDataOnHide is true, we remove the field and data from the model once the field is hidden.
+		if (this.options.deleteDataOnHide) {
```

Comment:
This could be pretty risky. Does a wizard form `destroy` the pages that aren't visible? If so, this would get called every time a page is changed. Same with when we show success messages potentially or anytime we take down the form.

I think it is safer to actually do this sort of thing when visiblity changes based on when the function is evaluated. 

Additionally we also need to be able to reset this to the initial value when visibility is no longer false. So on show, we want to set the initial value, otherwise we could lose the initial value by enabling and then disabling the field